### PR TITLE
SearchInput 

### PR DESCRIPTION
**Pull request by ThoJah**

### CODE REVIEW

**Comment by ThoJah** [comment]
SearchInput Issue complete and ready for review

File: src/components/SearchInput.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import { ChangeEvent, useState } from "react";
+import { MagnifyingGlassIcon } from "@heroicons/react/24/solid";
+import "../index.css";
```
Comment (NoHara42) [inline src/components/SearchInput.tsx:3]: remove this and place the import higher up in the hierarchy for your project. i.e. in main.tsx or app.tsx

File: src/stories/SearchInput.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+export default {
+  title: "Home/SearchInput",
+  component: SearchInput,
+} as ComponentMeta<typeof SearchInput>;
+
+const Template: ComponentStory<typeof SearchInput> = (args) => (
+  <SearchInput {...args} />
+);
+
+// export const Primary = Template.bind({});
```
Comment (NoHara42) [inline src/stories/SearchInput.stories.tsx:16]: please remove large blocks of unused code before committing

File: src/stories/SearchInput.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+// Primary.args = { type: "primary", label: "default label" };
+
+// export const Secondary = Template.bind({});
+// Secondary.args = { type: "secondary", label: "default label" };
+
+export const Primary = () => {
+  // Sets the hooks for both the label and primary props
+  const [value, setValue] = useState("Secondary");
```
Comment (NoHara42) [inline src/stories/SearchInput.stories.tsx:24]: why are you using reacts useState inside a story?

**Review by NoHara42 — CHANGES_REQUESTED** [review]
See comments, let's chat about this

**Review by NoHara42 — APPROVED** [review]
LGTM

**Pull request merged**